Podfile

```diff
@@ -7,6 +7,7 @@ inhibit_all_warnings!
 target :Awful do
   pod 'AFNetworking', '2.5.0'
   pod 'ARChromeActivity', '1.0.4'
+  pod 'AsyncDisplayKit', '1.1.1'
```

Comment:
Just FYI, ASDK is updated at a pretty decent clip. ASEditableTextNode was just released onto master (probably going to be tagged soon), which is pretty neat.